QuizApp/Login/LoginViewController+Design.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+import SnapKit
+
+extension LoginViewController: ConstructViewsProtocol {
+
+    func createViews() {
+        gradientLayer = CAGradientLayer()
+        gradientLayer.frame = view.bounds
+        view.layer.addSublayer(gradientLayer)
+
+        logoLabel = UILabel()
+        view.addSubview(logoLabel)
+    }
+
+    func styleViews() {
+        gradientLayer.type = .axial
+        gradientLayer.startPoint = CGPoint(x: 0.5, y: 1)
+        gradientLayer.endPoint = CGPoint(x: 0.5, y: 0)
+        gradientLayer.colors = [UIColor.darkerPurple.cgColor, UIColor.lighterPurple.cgColor]
+
+        logoLabel.font = UIFont.sourceSansPro(ofSize: 32, ofWeight: SourceSansProWeight.bold)
```

Comment:
You can omit `UIFont`  - [style guide](https://github.com/fiveagency/ios-swift-guide/tree/master/Code%20Style#type-inferred-context).